eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -252,9 +264,9 @@ stages:
 
         - script: |
             set -e
-            twine upload --repository ${{parameters.DevFeedName}} --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{artifact.safeName}}/*.dev*.whl
+            twine upload --repository $(DevFeedName) --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{artifact.safeName}}/*.dev*.whl
```

Comment:
How does twine find the devfeed by name? That must be configured somewhere.

Reply:
Nevermind I now see the TwineAuthenticate step before this that is DevOps hooking up this repository by name for twine.